portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,35 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+    const fun1 = "I'm a big fan of Marvel and DC!";
+    const fun2 = "I enjoy various of sports and traveling"
+    const fun3 = "I love drawings and calligraphy"
+    const fun4 = "I love to try new things and meet with new people"
+    const fun5 = "Welcome to contact me anything to have some random chat!"
+    const greetings =
+        [fun1, fun2, fun3, fun4, fun5];
+  
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+  }
+  
+async function showServerTime() {
+    const responseFromServer = await fetch('/date');
+    const textFromResponse = await responseFromServer.text();
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    const dateContainer = document.getElementById('date-container');
+    dateContainer.innerText = textFromResponse;
 }
+
+// async function myRespond() {
+//     const responseFromServer = await fetch('/information');
+//     const textFromResponse = await responseFromServer.text();
```

Comment:
Once you update your information/ servlet, play around with this code in the Chrome Dev Tools console. It may be that you need to use a method other than `.text()` (not sure!).